pkg/daemon/daemon.go

```diff
@@ -381,7 +395,7 @@ func rebootNode() {
 	// as systemd will time out the stop invocation.
 	cmd := exec.Command("systemd-run", "--unit", "sriov-network-config-daemon-reboot",
```

Comment:
would it make sense to expose this command through configuration? In this way not only you would be able to disable node reboot completely, but others could adjust how the reboot is achieved. (Perhaps because the platform doesn't use systemd, or some additional steps should be executed before calling to 'reboot'.)

Of course, this is up to maintainers of the project to decide how far they want to go.

Reply:
From my point of view it would make sense, but I would treat it as a separate pr / issue.